projects/driver_controls/inc/driver_io.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+// Driver IO pin definitions
+
+#define DRIVER_IO_POWER_SWITCH_PIN          0
+#define DRIVER_IO_DIR_SELECT_PIN_FORWARD    2
+#define DRIVER_IO_DIR_SELECT_PIN_REVERSE    3
+#define DRIVER_IO_CRUISE_CONTROL_PIN        4
+#define DRIVER_IO_CRUISE_CONTROL_INC_PIN    5
+#define DRIVER_IO_CRUISE_CONTROL_DEC_PIN    6
+#define DRIVER_IO_TURN_SIGNAL_PIN_RIGHT     7
+#define DRIVER_IO_TURN_SIGNAL_PIN_LEFT      8
+#define DRIVER_IO_HAZARD_LIGHT_PIN          9
+
+#define DRIVER_IO_GAS_PEDAL_PIN             0
+#define DRIVER_IO_MECHANICAL_BRAKE_PIN      1
+
+// Digital address definitions
+
+#define DRIVER_IO_POWER_SWITCH              { GPIO_PORT_C, DRIVER_IO_POWER_SWITCH_PIN }
```

Comment:
I feel like this would make more sense if you flipped the order of address definitions vs. pins.

So define your port/pin address first, then make a macro that pulls the pin from the port/pin address.